eng/Subsets.props

```diff
@@ -39,10 +39,20 @@
   <PropertyGroup>
     <DefaultSubsets>clr+mono+libs+installer</DefaultSubsets>
     <DefaultSubsets Condition="'$(TargetOS)' == 'iOS' or '$(TargetOS)' == 'Android' or '$(TargetOS)' == 'tvOS'">mono+libs+installer</DefaultSubsets>
-    <DefaultSubsetCategories Condition="'$(TargetOS)' == 'WebAssembly'">mono+libs</DefaultSubsetCategories>
+    <DefaultSubsets Condition="'$(TargetOS)' == 'WebAssembly'">mono+libs</DefaultSubsets>
   </PropertyGroup>
 
+  <!-- Init _subSet here in to allow RuntimeFlavor to be set as early as possible -->
```

Comment:
```suggestion
  <!-- Init _subset here in to allow RuntimeFlavor to be set as early as possible -->
```

Reply:
Same - can that happen in a follow up PR?

Reply:
of course, those were just nits.